src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,185 @@
+"""
+Automation for default debug level
+
+:requirement: SSSD - Default debug level
+"""
+
+from __future__ import annotations
+
+import pytest
+from pytest_mh.ssh import SSHProcessError
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__check(client: Client):
+    """
+    :title: Check default debug level when sssd started successfully
+    :setup:
+        1. Clear logs and cache
+        2. Start SSSD with default debug level
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. "Starting with debug level = 0x0070" is in each file and
+            if log contains more than one line, log message with number "0x3f7c0" is stored
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    for file in [client.sssd.logs.monitor, client.sssd.logs.domain(), client.sssd.logs.nss, client.sssd.logs.pam]:
+        log_str = client.fs.read(file)
+        assert "Starting with debug level = 0x0070" in log_str, f"Log file has wrong format: {log_str}"
+
+        if len(log_str.split("\n")) > 1:
+            assert "(0x3f7c0)" in log_str, f"Log file has wrong format: {log_str}"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_default_debug_level__check_with_login(client: Client, provider: GenericProvider):
+    """
+    :title: Successful login with default debug level doesn't generate any logs
+    :setup:
+        1. Add user to SSSD, set its password
+        2. Add fallback_homedir (generates extra logs on
+            user auth if not specified)
+        3. Clear cache and logs
+    :steps:
+        1. Start SSSD with default debug level
+        2. Check total log size before user authentication
+        3. Authenticate with user
+        4. Check total log size after authentication
+    :expectedresults:
+        1. SSSD started successfully
+        2. Checked correctly
+        3. User is authenticated
+        4. Log sizes before and after are the same
+    :customerscenario: False
+    """
+
+    def log_length() -> int:
+        try:
+            output = client.fs.wc("/var/log/sssd/*", lines=True).stdout
+            return int(output.split()[0])
```

Comment:
This does check only first file in `wc -l` output. I think you want to check all files?

It might be better to make a copy of /var/log/sssd [1] and then run diff on the two folders.

[1] https://pytest-mh.readthedocs.io/en/latest/api/pytest_mh.utils.fs.html#pytest_mh.utils.fs.LinuxFileSystem.copy

Reply:
Thanks, fixed